web/frontend/src/lib/base-marker.txt

```diff
@@ -1 +1 @@
sjInmf2ttfba32XvtddaO0d2UQCmAdOBGfHvycDwmEbHfwGuAx/Hf68DF4B24DTQFv++lqDudSOXtgJlGAHMARbE9AgQeeR/FthbRh975N2wGAWsAw4Cd4DehOgOcABYG+vwf4UIWALsAjpJzui1qDPWZTF+n7jMIQKWAkdJ3+i16J/A00A+kA1SQR74LvA+6Ru4XjoDrGEQDMRs4B+kb1ArvQfM9W6VBPAgsA3oJn0julIPsB0Y59VCAdEKXCF9w/mmj4BveLSTd+SBnyN3TNrGCvk0bAOGerKZN0wBjpC+gZKid4BP+zCcD8wGLpG+UZKmK8ATrsZzdUV8FfgT4kYIgW7gGHCSfp/OWe71/cC9vqHPIP6jacBMxKXRFEi/EvBNYE8g/vfFSuAW/u+sC8AWZNE20oOehZjXVuBiAH1vASs86KnCSvxOMW8gU70FhHUHNAELgdfw6wbpJsFBmA/c9KR4BzKrmJCU8mUYC/wMuFqnrgPRbeDJ0Eo/jhjNh7IbkQVb2ngQ2ITo5NqvIjIpCYIpwGUPSu4HHg6lpAM+h7jFXft3iQBT1KG4z/M7gR+QrU2gSuSA9bi/Yg8DQ3wqttVRoXPAF30qFBiPIh5Rlz5v9qVMK27uhT34mU4mjQKwD3u/e4BlrkqMxc2xtosM+k0UGAbsxt7/j4AxLgr8zkH4rxkcW3wR8BvsdnjVKngu9lfPXwi3/E8DEfI0W2zRjcFnlAdOGAXuwf9r5yFgFbJo+yvygfwvMne/Hf8+E/9vG7JSf8izDsOwfxOOo7wh1xoFtePvg9sCPIfbluYRZFrZ4kmnAva97WfqFZI3CulEPI+umIRMe68bdKhFJcTBN9GDfrOwrRPaqfMpWG1g3ossslwwBNiAGMuX4asNxPO4L5LWG+WvHIhxhMTGaBnvx22FOx2JQAhl+Eo6juwXWJEDDhnknmIAOy0xML2Nm2/nKfw4+LRUBJY76P15oMsgd9H9mFoWHRsdOvF9ko0JraQ7SGyqFb80yNxZi1kB2RjRMOvAPsP4nkH5ULTe2IcW9E9vJzUCgtcZFP+FUfGnSPfOr/YktBr7sskgb001Rlpf+A3gkwaFp5POO38gKgJTDf0Zj357861KJgX0d+R2g7JDSXa2o6Vj2Kaov1fK6aJiwbrMoOwCg6IbDHKSpp8Y+vU1g5wl5Qy2KC++gN7TOZGwiyxf1IG8VjRoAj5QytkM/UacpxT4R8RTqsFP6U+qyzJGoH8KupEANQ3u2ryA3u38daWwFvz6dkJTCf30ulUpoxtojpBZicaN0A38TancauATymvSxHD0gVb7EdvUiwiY1jcAGhxDgpk0+LayfRbwtLL9VWSGp8EMywCcULYfj7hwGw2PoV/nnFS2Nz0B7cr2XyHbsUC1kAO+rLxGa5vpEfpNijZl+0aKB6rEHGX708r2kyKgWXnReWX7Gcr2WYJW9/PK9iMtA1BUtv+ssn2WoNVdWxCkOYkBaORaDFrdO5TtR0To04tKWiHK9lmC9ubUDkDzYIhca2hEhL+jtfyzBPUdreUfoX+nawOvGrkwknbFbxoA7R2qHYD3le2zhDPK9gVl+1KE/jGbomyvXbhlCVrdtbYpRsC/lBdpA5oOK9tnCW8r22vdOhcj9KOsFfIm4v9uNPQideU00NqmLULvv/iCsv2HwLvKa7KAdxDdNdDapi1C78F7FP0K8TVl+yxAq/MopDaFBm0gsxrtluRSpaAWGmND3mVLcrlSxt0tySL6jYT5yvZXgFeU16SJlxGdNdDa5ARlM1BtWMpF9DlgjRKWUkSf3pQH/q2Us6mcgSUwa6FSSYAXDHKSph8b+mUJzFpczqCAPtbd8mEdgmzqp23kWnQUW2jiDqWcLqq4LQ4omXRiy0SchjzmaRu7kq5h2zyagD44981qjCyZkdbkjOVkLzzdWlZgs0Fe1YxJS4JGCXsqviUfIQT1IKWVLfCaoAG2bPBNVTnVh3Wkn6K01kF/SxWZP9yP4WIDw9tIsSMrWknnm3ANt2omMwmQpJdDUim1TA/iFnw1lWRnR0dxi9bIIfGxWrknqcNOq4ydsia69WEIEsIeMn2piMzzXRO1f2SUX1fAbxO2alE38RMDOh5J//S5ai4hsxUfBTwew1YvtQ2F92CNQUAvsv2o3ZarhRbgWWRDx1I2pwfZUPkh/qozjkIq91ps851qDGu9j/KID1/rXgVZZCxG7hJfGIcE+c5BwgWnINW8+iI0SkhFx3PI/sZhJBPxkkcdHgDeQB+wC/J9exxd/sDdCyyj/TqDr2CTtXSZqWBTH35rFNqLlPkaDIFfTYgr3cUOZoxBCs9Zhe9GKk01KoYhCYnW/l/GQ7GoZbiVrdyHvw9zkhiFfEes/e5Bv3NYE5aqIOV0Hn2yQ5qYhfuxWy/6VGgoEiXgotBNZLGW5XSlHLLIcj0X4W08ly4GKUjt46iSQ0ixo6xhJjb3QiV9CHwqlJKz8eM460Jea05VZT1hDOLVtDjWKqlIAhmh8/B3gEMJqfPpo5qhFuOQAxyu1anrQHQLOVMnEazA7xEmnUjJlycJu4BrQjbQd+D/CJNvBdS7KlYQ5hCfD4BfIfsEoz3oOTrm9RL60JF673yz8V1nJAuBPxP2GKsTSAmA9pjOIe/aq/TnNoxA5u0jET9R31HoM2MK9VR1IMdY7Q3Evy7MQr78vu+srNN/yFAJhsnA30nfKEnREfTJGMGRR2YVg+EI21qU2cM8y7EMNwdeVukyHn07oTGawXeg81ivFkoIs2jsY26PA1/ybpWE0YSE4rWTvkHrpTZkD3cw7eoRIe/Qo6Rv4Fp0CilVlg9kg0wgh0SH7UQfixqCbiDhgovItqs8CApICMxb+PFG1ktdscxnSHm3LksjPhyJHlgQ0yP43dQ/i7gM9iInPWnrQARBlgagEs30H00+I/49CfH79Pl+yuOC+nxDJaS0cjsSI9R3FLq2JEMi+B+uTYhludzIjgAAAABJRU5ErkJggg==
```

Comment:
For my own edification, how are these text files created and related to the png?

Reply:
Good q! These are just base64 encoded string representations of each image file. Unfortunately right now we don't have a good strategy for serving static assets like images within the multiple environments that remote control exists in (local, standalone, embedded in app) and since these are small PNGs, it's a bit easier and safer to just encode them as strings so that they're bundled directly into JS code. The actual PNGs here are just for reference and for when we do come up with a good enough strategy to serve them as separate files, since that's the better long term solution.

Reply:
Do you have a script that converts them to text?

Reply:
Just added a a readme that gives an example for how to do the transform. I'm reluctant to add anything more since this is a stopgap solution that was needed to fix prod icon rendering while we figure out asset management long term. It would become a pretty nasty antipattern if we continued to scale it.